sale_outplacement/models/project.py

```diff
@@ -42,37 +42,49 @@ class ProjectTask(models.Model):
     @api.model
     def init_joint_planning(self, outplacement_id):
         for task in self.env["res.joint_planning"].search([], order="sequence"):
-            stage_todo = self.env.ref(".".join([xmlid_module, 'stage_todo']))
-            stage_optional = self.env.ref(".".join([xmlid_module, 'stage_optional']))
-            stage_done = self.env.ref(".".join([xmlid_module, 'stage_done']))
-
-            if not stage_todo:
-                stage_todo = self.env['project.task.type'].create({'name': 'To Do'})
+            try:
+                stage_todo = self.env.ref(".".join([xmlid_module, 'stage_todo']))
+            except:
```

Comment:
Do we know the actual exception here, better to use explicit excepts instead of bare ones